index.html

```diff
@@ -3,63 +3,78 @@
 <head>
     <meta charset="UTF-8">
     <title>Title</title>
+    <link rel="stylesheet" href="css/style.css">
 </head>
 <body>
-<form action="#" method="get" enctype="multipart/form-data">
-    <div>
-        <p>Username</p>
-        <input type="text" placeholder="Username" name="username">
-    </div>
-    <div>
-        <p>Password</p>
-        <input type="password" placeholder="Password" name="password">
-    </div>
-    <div>
-        <p>Phone</p>
-        <input type="tel" placeholder="Phone" name="phone">
-    </div>
-    <div>
-        <p>Gender</p>
-        <div>
-            <input checked type="radio" name="radio1" value="male">Male
-        </div>
-        <div>
-            <input type="radio" name="radio1" value="female">Female
+<header class="header">
+    <div class="container">
```

Comment:
Please move all styles regarding header to separate css file. 
Name it **_header.css_** and add links to it in both html files.